### ISSUE TITLE
Declare Strict Types

### ISSUE THREAD

**Issue by JustSteveKing** [laravel-shift/workbench-issues#69]
**What would you like to do?**
I would love to be able to have a workbench task that adds `declare(strict_types=1)` at the top of every file in my Laravel application. This is currently something I do manually each time I touch a file, but having this automated would be a lot better!

Having this as a selectable dropdown of different strictness levels would most likely make this more usable for everyone

**Before/After code snippets**

### Before

```php
<?php

namespace App\Models;

class User extends Model
{
    // usual code goes here
}
```

### After

```php
<?php

declare(strict_types=1);

namespace App\Models;

class User extends Model
{
    // usual code goes here
}
```

**Comment by jasonmccreary** [comment]
Easy peezy lemon squeezy. 🍋 

I'll get this added this week.

**Comment by jasonmccreary** [comment]
This is available in 0.9.0.
